src/btwa_api/app/api/endpoints.py

```diff
@@ -1,44 +1,70 @@
+from typing import List
+
 from fastapi import APIRouter, Depends, HTTPException
 from fastapi.security import HTTPBasicCredentials
+from pydantic import BaseModel
 
-from ..db.database import Database
+from .utils import get_or_404
+from ..db.firebase.firebase import Firebase, get_firebase
+from ..db.sql.database import Database
 from ..db.utils import get_db
 from ..security import check_handler_auth, security
-from starlette.responses import JSONResponse
 
 router = APIRouter()
 
 
-@router.get("/register/{guid}")
-def register(guid: str,
+class User(BaseModel):
+    fuid: str
+    status: str
+
+
+@router.get("/get-user/{phone}", response_model=User)
```

Comment:
consider passing PHONE argument as POST data rather than GET URL argument. phone number might be considered a PI and thus should not be logged to basic webserver logfiles (possibly processed with logging data analytic services)

if {phone} is real phone number, it might be considered a PI. If argument is passed within GET request URI it will be stored in logs possibly processed by some